libmamba/src/core/link.cpp

```diff
@@ -164,9 +165,10 @@ namespace mamba
         std::ofstream out_file = open_ofstream(script_path);
 
         fs::u8path python_path;
-        if (m_context->has_python)
+        if (m_context->python_params().has_python)
         {
-            python_path = m_context->relocate_prefix / m_context->python_path;
+            python_path = m_context->prefix_params().relocate_prefix
```

Comment:
Does the prefix params have to be re-evaluated here or can we get them up in the function and read that? (same question for following functions, where it's relevant)

Reply:
The parameter is just used here, so I did'nt capture it at the beginning of the function to avoid adding code. Further refinement will be to remove the usage of `m_transaction_context` as much as possible in the implementation of the link related classes and have the required parameters passed as function parameters, but I think that deserves itw own PR.